src/interface/graphql/modules/key-result/check-mark/key-result-check-mark.node.ts

```diff
@@ -29,9 +29,16 @@ export class KeyResultCheckMarkGraphQLNode implements GuardedNodeGraphQLInterfac
   })
   public readonly keyResultId!: string
 
-  @Field(() => ID, { complexity: 0, description: 'The ID of the user this check mark belongs to' })
+  @Field(() => ID, { complexity: 0, description: 'The ID of the user who created this check mark' })
   public readonly userId!: string
 
+  @Field(() => ID, {
+    complexity: 0,
+    nullable: true,
```

Comment:
As we've mentioned, this field can't be null. If no user send was declared as assigned, we should use the user of the request